### ISSUE TITLE
About the released training code.

### ISSUE THREAD

**Issue by BIGJUN777** [yangxy/GPEN#80]
Hi, there. Thank you for releasing the training code.

- I found you adopt smooth L1 loss as the pixel-wise constraint. Does smooth L1 loss work better than L1 loss in this task?
- How long did you train the model of your released version to get competitive results? As I know, the Path Regularization for G net is really time-consuming.
- Would you share some competitive visualization results of the released version here so that we can check our training results?

Thanks.

**Comment by yangxy** [comment]
1. Almost the same perceptually. No worries. Please focus on the large picture as BFR is perceptual-oriented. 
2. Yes, you are right. For me, it would take at least 4 days to obtain a quite good model using a V100 machine with 4 GPUs. The model could keep improving with more epochs.
3. You need to find it out yourself.

**Comment by BIGJUN777** [comment]
Cool, Thank you for your quick reply.
